app/helpers/application_helper.rb

```diff
@@ -599,9 +599,9 @@ def concurrent_events(event)
   end
 
   def speaker_selector_input(form)
-    users = User.active.select(:id, :name, :username, :email).sort_by {|user| user.name}
+    users = User.active.pluck(:id, :name, :username, :email).map { |user| [user[0], user[1].blank? ? user[2] : user[1], user[2], user[3]] }.sort_by { |user| user[1] }
```

Comment:
user[1] always has a value because of the `user[1].blank? ? user[2] : user[1]` in .map, which is the equivalent of User#name (https://github.com/openSUSE/osem/blob/master/app/models/user.rb#L83)